src/assets/styles/base/_base.scss

```diff
@@ -121,37 +121,44 @@ blockquote {
 .section {
```

Comment:
There is some inconsistency with the way ordered and unordered lists are styled:

![image](https://github.com/inclusive-design/idrc/assets/564762/4bc3c5e2-d60b-401c-91bb-a4314f2e1ce7)

Source:  https://idrc.ocadu.ca/projects/reducing-barriers-to-virtual-healthcare/

1. Ordered lists appear outside the content margin - they should be horizontally aligned the same way as unorderd lists.
2. Text wrapping with an unordered list item appears different than the ordered list. I suspect this has something to do with the way the ::before element is being positioned for ULs?

Reply:
I made fixes to address:

1. Ordered lists now aligned horizontally like unordered lists.
2. Text wrapping for ordered list to follow unordered list. As you suspected, this was happening because ::before element is being positioned for UL. List have list-style-position property, and setting that to "inside" makes the wrapping to place words under the numbers.

I also removed different styles for nested unordered list items. Could you review the changes when you have time please? Changes can be also viewed on https://deploy-preview-700--idrc.netlify.app/repertoire/